action/protocol/execution/evm/evmstatedbadapter_test.go

```diff
@@ -61,6 +62,36 @@ func TestAddBalance(t *testing.T) {
 	require.Equal(0, amount.Cmp(big.NewInt(80000)))
 }
 
+func TestEmptyAndCode(t *testing.T) {
+	require := require.New(t)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	testutil.CleanupPath(t, testDBPath)
+	defer testutil.CleanupPath(t, testDBPath)
+	ctx := context.Background()
+	cfg := config.Default
+	cfg.Chain.ChainDBPath = testDBPath
+	cfg.Explorer.Enabled = true
+	sf, err := factory.NewFactory(cfg, factory.InMemTrieOption())
+	require.NoError(err)
+	require.NoError(sf.Start(ctx))
+	defer func() {
+		require.NoError(sf.Stop(ctx))
+	}()
+	ws, err := sf.NewWorkingSet()
+	require.NoError(err)
+	mcm := mock_chainmanager.NewMockChainManager(ctrl)
+	mcm.EXPECT().ChainID().Times(3).Return(uint32(1))
+	addr := common.HexToAddress("02ae2a956d21e8d481c3a69e146633470cf625ec")
+	stateDB := NewStateDBAdapter(mcm, ws, 1, hash.ZeroHash32B, hash.ZeroHash32B)
+	stateDB.CreateAccount(addr)
+	require.True(stateDB.Empty(addr))
+	stateDB.SetCode(addr, []byte("0123456789"))
+	require.True(bytes.Equal(stateDB.GetCode(addr), []byte("0123456789")))
+	require.False(stateDB.Empty(addr))
+}
```

Comment:
add an non-existing address b
require.False(stateDB.Empty(b)), this will assert that state.EmptyAccount